drivers/hmis/app/graphql/types/forms/form_identifier.rb

```diff
@@ -0,0 +1,55 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class Forms::FormIdentifier < Types::BaseObject
+    skip_activity_log
+    description 'FormIdentifier'
```

Comment:
Comment below is helpful but it would be nice to update the description too
```suggestion
    description 'Type representing one form Identifier, which collects all versioned FormDefinitions for the same identifier'
```